### PR TITLE
Fix segfault when bundled font are not found

### LINKED ISSUE #927: FontContext segfaults if no fonts can be found
screen shot 2016-08-23 at 3 33 12 pm

### PR DESCRIPTION
**Pull request by karimnaaji**
Fix #927

### CODE REVIEW

**Comment by tallytalwar** [comment]
@karimnaaji I modified this. Can you give this a look?
Thanks

File: core/src/text/fontContext.cpp [hunk 1]
```diff
@@ -46,13 +46,13 @@ void FontContext::loadFonts() {
             size_t dataSize;
             char* data = reinterpret_cast<char*>(bytesFromFile(DEFAULT, dataSize));
 
-            if (data) {
                 LOG("Loading default font file %s", DEFAULT);
```
Comment (karimnaaji) [inline core/src/text/fontContext.cpp]: @tallytalwar indentation is off here now

**Comment by karimnaaji** [comment]
@tallytalwar yeah like we spoke about this morning this is good.

File: core/src/text/fontContext.cpp (hunk 1 above)
Comment (tallytalwar) [inline core/src/text/fontContext.cpp]: Updated.

**Pull request merged**